build/package/Installer.RPM.targets

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project='Installer.RPM.props'/>
+
+  <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(CLIBuildDll)"/>
+  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(CLIBuildDll)"/>
+
+  <PropertyGroup>
+    <BuildRpmPackage Condition=" '$(IsRPMBasedDistro)' == True and '$(Architecture)' == 'x64' ">True</BuildRpmPackage>
+  </PropertyGroup>
+
+  <Target Name="GenerateRpms"
+          DependsOnTargets="TestFPMTool;BuildRpms;TestSdkRpm"
+          Condition=" '$(BuildRpmPackage)'=='True' "
+          Outputs="@(GeneratedInstallers)"/>
+
+  <Target Name="BuildRpms"
+          DependsOnTargets="GenerateSharedHostRpm"
+          Condition=" '$(BuildRpmPackage)'=='True' and '$(FPMPresent)'=='True' "/>
+
+  <Target Name="GenerateSharedHostRpm">
+    <PropertyGroup>
+      <RpmPackageVersion>$(SdkVersion)</RpmPackageVersion>
+      <InputRoot>$(OutputDirectory)/sdk</InputRoot>
+      <SdkInstallerFile>$(InstallerOutputDirectory)/$(DistroSpecificArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk)$(InstallerExtension)</SdkInstallerFile>
+      <RpmFile>$(SdkInstallerFile)</RpmFile>
+      <ManPagesDir>$(RepoRoot)/Documentation/manpages</ManPagesDir>
+      <ConfigJsonFile>$(RepoRoot)/packaging/rpm/dotnet-config.json</ConfigJsonFile>
+      <RpmIntermediatesDir>$(IntermediateDirectory)/$(RpmPackageName)/$(RpmPackageVersion)</RpmIntermediatesDir>
+      <RpmTestResultsXmlFile>$(RpmIntermediatesDir)/debian-testResults.xml</RpmTestResultsXmlFile>
+      <RpmInstalledDirectory>/usr/share/dotnet</RpmInstalledDirectory>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <rpmLayoutDirectory>$(RpmIntermediatesDir)/rpmLayoutDirectory/</rpmLayoutDirectory>
```

Comment:
Let's stay consistent with the naming of these properties. I prefer either Rpm or RPM at the beginning. And I see in some cases you used Rpm, and other just rpm, all lowercase.